source/loader/ur_lib.hpp

```diff
@@ -94,6 +97,9 @@ class __urdlllocal context_t : public AtomicSingleton<context_t> {
 #endif
 #if UR_ENABLE_SANITIZER
             ur_sanitizer_layer::context_t::getNames(),
+#endif
```

Comment:
you also need to add the layer info to `const std::vector<LayerData> layers` above.

Reply:
Thanks @pbalcer have done so